cpp/Result.hpp

```diff
@@ -424,6 +429,11 @@ namespace hsd
             return _initialized;
         }
 
+        explicit constexpr operator bool() const
```

Comment:
there is a `.is_ok()`, reason?

Reply:
To use in `if`s and boolean expressions. It's just quicker.